lightstep/resource_metric_dashboard_test.go

```diff
@@ -220,14 +226,19 @@ resource "lightstep_metric_dashboard" "test" {
 			},
 			{
 				Config: spansQueryDashboardConfig,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckMetricDashboardExists(resourceNameSpans, &dashboard),
-					resource.TestCheckResourceAttr(resourceNameSpans, "dashboard_name", "Acceptance Test Dashboard"),
-					resource.TestCheckResourceAttr(resourceNameSpans, "chart.0.query.0.display", "line"),
-					resource.TestCheckResourceAttr(resourceNameSpans, "chart.0.query.0.hidden", "false"),
-					resource.TestCheckResourceAttr(resourceNameSpans, "chart.0.query.0.spans.0.operator", "error_ratio"),
-					resource.TestCheckResourceAttr(resourceNameSpans, "chart.0.query.0.spans.0.query", "service IN (\"frontend\")"),
-				),
+				Check: func(state *terraform.State) error {
```

Comment:
why use a closure here? I don't see state being used

Reply:
I added this so I could set a breakpoint while debugging. Will remove!